cgmes/cgmes-conversion/pom.xml

```diff
@@ -34,6 +34,13 @@
                         </manifestEntries>
                     </archive>
                 </configuration>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>test-jar</goal>
```

Comment:
Can you explain why this is needed?

Reply:
sorry, that went without wanting. I will remove this change.
I needed to kept that to be able to use test code from conversion in some of my external modules, but I am splitting the test code from conversion that is reused in #2440, following the same schema done in #2370